python/paddle/fluid/contrib/utils/lookup_table_utils.py

```diff
@@ -80,48 +78,57 @@ def __get_prefetch_op_tuples(main_program):
     return prefetch_op_tuples
 
 
-def convert_dist_to_sparse_program(main_program):
-    if not main_program._distributed_lookup_table:
+def convert_dist_to_sparse_program(progarm):
+    """
+    WARNING: this function will only used for distributed training with distributed lookup table.
+    when we train model with distributed lookup table but want to do the local inference, we can use
+    this function to convert the train program with distributed lookup table to sparse lookup table.
+
+    :param program(Program): the program must be the trainer program, which will be get by the distribute transpiler.
```

Comment:
Docstring should follow: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Done.